src/nuget/Microsoft.Build.nuspec

```diff
@@ -58,7 +58,7 @@
         <dependency id="System.Text.RegularExpressions" version="4.1.0" />
         <dependency id="System.Threading" version="4.0.11" />
         <dependency id="System.Threading.Tasks" version="4.0.11" />
-        <dependency id="System.Threading.Tasks.Dataflow" version="4.6.0" />
+        <dependency id="System.Threading.Tasks.Dataflow" version="4.9.0" />
```

Comment:
This is harmless but I believe this file is dead; microsoft/msbuild#4618 to figure out if that's true.